test/unit/firebase-namespace.spec.ts

```diff
@@ -51,6 +51,7 @@ import {
 import { InstanceId } from '../../src/instance-id/instance-id';
 import { InstanceIdImpl } from '../../src/instance-id/instance-id-internal';
 import { ProjectManagement } from '../../src/project-management/project-management';
+import { ProjectManagementService } from '../../src/project-management/project-management-internal';
```

Comment:
Use ProjectManagement. Let's not introduce any `*Service` classes until we change the `FirebaseService` interface itself.